models/staging/stg_github_contributions.yml

```diff
@@ -8,10 +8,9 @@ models:
           - not_null
           - unique
       - name: user_login
```

Comment:
@ramonvermeulen : Could you try something ugly but potentionally nice?

```
- name: lower(user_login)
  tests:
    - accepted_values:
        values: "{{ var('author_string').split(',') | lower }}"
```

Or with backticks around the column name

Reply:
I tried this already (`{{ var('author_string').split(',') | lower }}`) but it will convert to a character by character lowercase array

Reply:
@JCZuurmond 
Update: works how I implemented it now, so no generic test required :)